### ISSUE TITLE
Updated Budgeting Process

### ISSUE THREAD

**Issue by pwensing** [pwensing/RAS-Governance#29]
- [ ] Pat: Capture current status in bylaws within this thread
- [ ] Pat & Jen Jen: Determine on what details will go in Bylaws vs. P&P
- [ ] Pat to take any Bylaws updates
- [ ] Jen Jen to take any P&P updates

**Comment by pwensing** [comment]
The budget is only mentioned in two locations.

In section 3.4: Vice President for Financial Activities:

> The Vice President for Financial Activities, with the assistance of the Financial Activities Board, shall prepare the Society budgets which shall then be submitted to the AdCom for approval before June 30th each year. The Vice President for Financial Activities shall serve as an ex-officio member of the AdCom without vote.

In Section 3.2: Treasurer:

> The Treasurer shall serve on the Financial Activities Board and work in strict cooperation with the Vice President for Financial Activities to help the Society to stay on budget.

**Comment by JenJenChung** [comment]
Agreed, I think that's a good breakdown between the Bylaws and P&P.

One thing that Aude noted in her last email was to allow FAB authority to transfer funds within boards (across initiatives) or across boards for small amounts (<=100K). I think that would need to be presented as a separate motion to AdCom, and if it passes, would then end up in the P&P. What do you think?

**Comment by pwensing** [comment]
I'm actually quite unclear on the order of some of these updates to P&P and Bylaws. For example, we, in the past, have voted to disband certain standing committees. Later we vote to approve Bylaws updates that include those updates. Other times, we have introduced governmental structure changes in the bylaws edits, and didn't include a separate motion for those changes. My sense is that either is fine, and that any passed motions will "in stone" even if they are not yet reflected in the bylaws. However, if there is a change to P&P / Bylaws that we think could be controversial, then we should have a separate motion on it in advance to avoid that one thing holding up the rest of the edits. 

Does that sound reasonable from a governance side? Or is there something you think we should be more careful with? 

Regarding the specific item about FAB having authority to transfer funds within boards, I do think this is something we should vote on at AdCom, since most of our existing financial motions are well below the $100K mark.

**Comment by pwensing** [comment]
@JenJenChung I pushed proposed updates in 8644c136a0375b77e41869d8fa3fe1a16c2957e3 -- please take a quick check and propose any update here. 

This does raise a constraint regarding sequencing in that we will want to put forward the P&P updates before the Bylaw updates.

**Comment by pwensing** [comment]
Great catch! I am bad at spelling :)

**Comment by pwensing** [comment]
From Lu -- here are the travel guidelines from IEEE ([link](https://www.ieee.org/content/dam/ieee-org/ieee/web/org/travel-expense-reimbursement-guidelines.pdf)).
